packages/core/docs/contributing/creating-changelog.md

```diff
@@ -0,0 +1,48 @@
+# Creating changelog
+
+To keep our changelog consistent, it is created automatically. 
+Now you don't need to worry about its structure and order of the information. 
+There are just a few steps to do in order to create the changelog. 
+
+## Steps
+
+1. Go to folder `/changelog`, create a `.js` file, and give it the same name as a number of your current issue from GitHub. For example `4523.js`
+2. Copy this code below, paste it inside your `.js` file and fill it in with your data and save it
+
+```js
+module.exports = {
+  description: "description of your changes",
+  link: "link to your issue",
+  isBreaking: true or false, //(depends if you applied some breaking changes),
+  breakingChanges: [ //leave it empty if there aren't any breaking changes
+    {
+      module: "affected module",
+      before: "how it was",
+      after: "how it is",
+      comment: "quick migration guide"
+    }
+  ],
+  author: "your name",
+  linkToGitHubAccount: "link to your github account",
+}
+```
+3. Check a release version in which your changes should appear. Let's assume it's `2.0.14`
+4. In your console run `yarn changelog --v 0.14.3` 
+5. And it's done! Your changes should appear in `changelog.md` and the `/changelog` folder is empty now
+
+## Changing default `in` and `out` paths
```

Comment:
This part is not relevant to contributing, lets comment it out and leave for a README of a package once we externalize it